addons/l10n_ec_sale/tests/test_sale_to_invoice_ec.py

```diff
@@ -0,0 +1,78 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields
+from odoo.fields import Command
+from odoo.tests import Form, tagged
+from odoo.tools import float_is_zero
+
+from odoo.addons.sale.tests.common import TestSaleCommon
+
+
+@tagged('-at_install', 'post_install')
+class TestSaleToInvoiceEC(TestSaleCommon):
+
+    @classmethod
+    @TestSaleCommon.setup_country('ec')
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
I'd also move some data in the setup, this is also good practice to make ready-to-use records of relevant models in the tests. In your case I think it make sense to create a Sale Order and a SRI payment method for example.
It will factorize a bit your code in tests below.